src/Extension/FluentExtension.php

```diff
@@ -1085,12 +1158,22 @@ public function updateLocalisationTabConfig(GridFieldConfig $config)
             new GroupActionMenu(GridField_ActionMenuItem::DEFAULT_GROUP)
         ]);
 
+        $copyToLocaleEnabled = $this->owner->config()->get('copy_to_locale_enabled');
+        $copyFromLocaleEnabled = $this->owner->config()->get('copy_from_locale_enabled');
+
         // Add each copy from / to
         foreach (Locale::getCached() as $locale) {
-            $config->addComponents([
-                new CopyLocaleAction($locale->Locale, true),
-                new CopyLocaleAction($locale->Locale, false),
-            ]);
+            if ($copyToLocaleEnabled) {
+                $config->addComponents([
+                    new CopyLocaleAction($locale->Locale, true),
```

Comment:
I'd recommend we use Injector to instantiate these action classes. I wouldn't block merge with this but it's generally a good idea for extensibility.